tests/test_order_Steinhardt.py

```diff
@@ -0,0 +1,90 @@
+import numpy as np
+import numpy.testing as npt
+import freud
+import unittest
+import warnings
+import util
+
+
+class TestSteinhardt(unittest.TestCase):
+    def test_shape(self):
+        N = 1000
+
+        box = freud.box.Box.cube(10)
+        np.random.seed(0)
+        positions = np.random.uniform(-box.Lx/2, box.Lx/2,
+                                      size=(N, 3)).astype(np.float32)
+
+        positions.flags['WRITEABLE'] = False
```

Comment:
For consistency with our other tests, I would remove this here and in other tests whether it's still present.